src/ert/gui/simulation/run_dialog.py

```diff
@@ -339,6 +349,21 @@ def _on_ticker(self):
         runtime = self._run_model.get_runtime()
         self.running_time.setText(format_running_time(runtime))
 
+        maximum_memory_usage = self._snapshot_model.root.data.get(ids.MAX_MEMORY_USAGE)
+        current_memory_usage = self._snapshot_model.root.data.get(
+            ids.CURRENT_MEMORY_USAGE
+        )
+
+        if maximum_memory_usage:
+            self.maximum_memory_usage.setText(
+                f"Maximum memory usage: {byte_with_unit(maximum_memory_usage)}"
+            )
+
+        if current_memory_usage:
+            self.current_memory_usage.setText(
+                f"Current memory usage: {byte_with_unit(current_memory_usage)}"
+            )
```

Comment:
This is extremely nitpick, but personally I think the logical order is first handle current, then the maximum. Could consider swapping the order here.